ddecode.c:169:21: warning: equality comparison result unused [-Wunused-comparison]
      d_note.length == ERR_INV_FORMAT;
      ~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~
ddecode.c:169:21: note: use '=' to turn this equality comparison into an assignment
      d_note.length == ERR_INV_FORMAT;
                    ^~
                    =
--- src/ddecode.c.orig	2018-11-05 05:42:00.000000000 -0600
+++ src/ddecode.c	2018-11-05 07:13:46.000000000 -0600
@@ -166,7 +166,7 @@
   d_note.length = ddecode_find_tag(d,"</body>");
   if ( d_note.length == -1)
     {
-      d_note.length == ERR_INV_FORMAT;
+      d_note.length = ERR_INV_FORMAT;
       d->position = pos;
       return d_note;
     }
